addons/sale_stock/views/sale_stock_portal_template.xml

```diff
@@ -64,11 +65,12 @@
                     </t>
                 </div>
             </div>
-            <t t-set="returns" t-value="sale_order.picking_ids.filtered(lambda picking: picking.picking_type_id.code == 'incoming')"/>
+            <t t-set="returns" t-value="delivery_orders.return_ids"/>
+            <t t-set="latest_returns" t-value="delivery_orders.return_ids.sorted('date', reverse=True)[:3]"/>
```

Comment:
Is it wanted to only display the last three returns rather than *all* returns for this SO?
If so, then it might be good to change `Returns` to `Last Returns` to match what's done for the delivery orders.

Reply:
On second thought, let's keep it simple, it's unlikely we will ever need to limit the returns